js/src/components/tree-select-control/control.js

```diff
@@ -36,6 +38,7 @@ const Control = ( {
 	onFocus = () => {},
 	onTagsChange = () => {},
 	onInputChange = () => {},
+	onControlClick = noop,
```

Comment:
What is the advantage of `noop` over `() => {}`

To consider using `= () => {},` to maintain consistency as well as no need of lodash dep. (I'm aware we are going to use it anyway in this component, but still)

Reply:
> What is the advantage of `noop` over `() => {}`

Usually the differences are not significant, but there are still some reasons I think a named and stable reference `noop` is more appropriate.

- More semantic. `noop` is short for no operations in computer science.
- Avoid unnecessary reference changes.
- Save memory a little bit.

> no need of lodash dep

Not sure if I missed any new updates, I thought this direction was not applied to this repo?

Reply:
Oh, I forgot the component were planing to be moved toward

Reply:
Hey @puntope, I'm going to merge this PR. I think we could make the uses of `() => {}` or `noop` consistent for all files of TreeSelectControl in a separate PR.